### PR TITLE
Integrate slack

### LINKED ISSUE #8: Connect to slack
# Overview
Connect Nostalgiabot to a slack instance. The scope of this ticket is just doing the initial minimal infrastructure for Nostalgiabot to be able to send and receive messages from the Slack instance

# Acceptance Criteria
- [x] Configurations for tokens are implemented
- [x] A dummy `hello` endpoint/service exists where a Slack user can message Nostalgiabot to get a "hello"

## Notes
- This was technically issue #4 on the old Trello numbering scheme

### PR DESCRIPTION
**Pull request by bellkelly**
Add Slack Integration to the Flask app.

- [x] Created an .env.example file to provide the slack bot tokens.

- [x] Created a `SlackBot` class to house the properties and methods for interacting with Slack

- [x] Created a slack_events file to hold the event handlers (Should this be a class?) 

refs #1 
[trello ticket](https://trello.com/c/eqYcLRXL/4-connect-to-slack)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,28 @@
 > \- Martin Petkov
 
 Nostalgiabot II is the sequel to Nostalgiabot, originally written by Martin Petkov.
+
+## Steps to Create Slack Bot
+
+1. Create a new .env file in the root directory of the project using .env.example as a template
+   ```
+   cp .env.example .env
+   ```
```
Comment (revangel) [inline README.md]: [*Nit*] There's some training whitespaces (lines 14, 15, 17, 22, 30)

**Comment by revangel** [comment]
# DevQA (+1)

image

- [x] Readme instructions were correct
- [x] NB responds when mentioned

**Comment by revangel** [comment]
# Code (+1)

- Just that nit in the README

File: README.md (hunk 1 above)
Comment (bellkelly) [reply README.md]: Nice catch! I'll fix that.

**Pull request merged**